README.md

```diff
@@ -44,7 +44,9 @@ The following features are implemented within this package.
 * [Validation](/Hackney.Core/Hackney.Core.Validation/README.md)
 * [Validation.AspNet](/Hackney.Core/Hackney.Core.Validation.AspNet/ReadMe.md)
 * Testing
+  * [Hackney.Core.Testing.DynamoDb](/Hackney.Core/Hackney.Core.Testing/Hackney.Core.Testing.DynamoDb/README.md)

```

Comment:
Has the Dynamodb read me been converted to have it own read me (don’t see it in this PR…)

Reply:
No. This is the DynamoDb _testing_ package documentation, which did already exist but which didn't have a link in the root level readme.